lib/fedex_rest_api/tracking_object.rb

```diff
@@ -0,0 +1,25 @@
+require "fedex_rest_api/base"
+
+class FedexRestApi::TrackingObject
```

Comment:
Nit, but would `FedexRestApi::TrackingRequest` or `FedexRestApi::TrackingRequestObject` be any more clear?

Reply:
I'm going to let this marinade a bit long. It's _making_ a request but the return is the tracking _data_. I'll keep it in mind as I roll out the similar changed to Address and come back to update it here too if it still seems weird.